config.js

```diff
@@ -232,8 +232,14 @@ var config = {
     // Sets the preferred resolution (height) for local video. Defaults to 720.
     // resolution: 720,
 
-    // Specifies whether the raised hand will hide when someone becomes a dominant speaker or not
-    // disableRemoveRaisedHandOnFocus: false,
```

Comment:
You should not delete this, but deprecate it. Check the other options that have DEPRECATED.  in the comment.
You should also make sure that the old config works in parallel with the new one. That ensures that we are not breaking experience for people with the old config set.